pkg/ds/daemon_set_test.go

```diff
@@ -33,9 +33,9 @@ func waitForNodes(
 	ds DaemonSet,
 	desired int,
 	desiresErrCh <-chan error,
-	changesErrCh <-chan error,
+	dsChangesErrCh <-chan error,
 ) int {
-	timeout := time.After(1 * time.Second)
+	timeout := time.After(2 * time.Second)
```

Comment:
My test failed due to a timeout issue once, so I'm a little paranoid it might happen in the future.
It doesn't timeout using one second anymore though.